src/actions/graph.js

```diff
@@ -50,6 +76,7 @@ export function calculateGraph(graphState) {
     return updateGraph(graphState);
   }
   return async function(dispatch) {
+    dispatch(toggleIsLarge(false));
```

Comment:
There's still a bunch of leftover unnecessary actions & reducers in this - I assume you just haven't gotten around to removing them all yet?

Reply:
This has been deleted in latest commit